Platform/fr.lproj/Localizable.strings

```diff
@@ -125,7 +125,7 @@
 // UTMVirtualMachine.m
 "Suspended" = "Suspendue";
 "Stopped" = "Arrêtée";
-"Starting" = "Déamarre";
+"Starting" = "Démarre";
```

Comment:
I don't know if we would have enough room for this variant, but it seems more appropriate: `Démarrage en cours`